posthog/queries/trends/trends_event_query.py

```diff
@@ -65,6 +66,7 @@ def get_query(self) -> Tuple[str, Dict[str, Any]]:
             for column_name in sorted(self._column_optimizer.person_on_event_columns_to_query):
                 _fields += f', {self.EVENT_TABLE_ALIAS}."{column_name}" as "{column_name}"'
 
+        if self._using_person_on_events and groups_on_events_querying_enabled():
```

Comment:
Not sure what you mean here. I can't just condition on `groups_on_events_querying_enabled()` because _using_person_on_events defaults to false and there legit might be cases where it's invalid..